File: milliseconds/net9/Models/Requests/MillisecondRequest.cs
using System.Text.Json.Serialization;

namespace net9.Models.Requests;

public class MillisecondRequest
{
    [JsonPropertyName("year")]
    public int Year { set; get; }
    [JsonPropertyName("month")]
    public int Month { set; get; }
    [JsonPropertyName("date")]
    public int Date { set; get; }
    [JsonPropertyName("hour")]
    public int Hour { set; get; }
    [JsonPropertyName("minute")]
    public int Minute { set; get; }
    [JsonPropertyName("second")]
    public int Second { set; get; }
}

File: milliseconds/rust/src/models/requests/millisecond_request.rs
use serde::Deserialize;


#[derive(Debug, Deserialize)]
pub struct MillisecondRequest {
    #[serde(rename = "year")]
    pub year: i32,
    #[serde(rename = "month")]
    pub month: u32,
    #[serde(rename = "date")]
    pub date: u32,
    #[serde(rename = "hour")]
    pub hour: u32,
    #[serde(rename = "minute")]
    pub minute: u32,
    #[serde(rename = "second")]
    pub second: u32
}